SPEC.md

```diff
@@ -364,7 +366,7 @@ The schema validator will ensure that the keys conform to these constraints.
 
 ### Image Manifest Schema
 
-JSON Schema for the Image Manifest
+JSON Schema for the Image Manifest (app image manifest, ACI manifest)
```

Comment:
thanks, this is definitely useful. Maybe we should call it out earlier in the spec when we first define "image manifest"?